LICENSE.md

```diff
@@ -1,5 +1,5 @@
-Copyright (c) 2011-2020, Sideway Inc, and project contributors  
-Copyright (c) 2011-2014, Walmart  
+Copyright (c) 2011-2022, Sideway Inc, and project contributors
```

Comment:
Sideway copyright should be locked at 2011-2020 and project contributor moved to its one line like was done for Walmart.

Reply:
Good call, thanks for catching that.  Fix is here: https://github.com/hapijs/hoek/pull/378